src/Microsoft.ML.Data/Transforms/FeatureContributionCalculationTransformer.cs

```diff
@@ -218,7 +218,7 @@ public Mapper(FeatureContributionCalculatingTransformer parent, DataViewSchema s
                     throw Host.ExceptSchemaMismatch(nameof(schema), "input", _parent.ColumnPairs[0].inputColumnName);
                 _featureColumnType = schema[_featureColumnIndex].Type as VectorType;
                 if (_featureColumnType == null || _featureColumnType.ItemType != NumberDataViewType.Single)
-                    throw Host.ExceptSchemaMismatch(nameof(schema), "feature", _parent.ColumnPairs[0].inputColumnName, "vector of float.", _featureColumnType.ItemType.ToString());
+                    throw Host.ExceptSchemaMismatch(nameof(schema), "feature", _parent.ColumnPairs[0].inputColumnName, "Vector of Single", _featureColumnType.ItemType.ToString());
```

Comment:
>Vector [](start = 120, length = 6)

While `Single` is appropriate, I think we generally call these `vector` in the context of a larger sentence.

So I would say "I want a vector of Single," I would not say "I want a Vector of Single."

Reply:
I try to figure out which one looks better, and come to conclusion what lower case vector is more appealing for me, but I guess i miss capital case in some places.

---
In reply to: [268028098](https://github.com/dotnet/machinelearning/pull/3046#discussion_r268028098) [](ancestors = 268028098)

Reply:
I agree, thank you @Ivanidzo4ka

---
In reply to: [268257594](https://github.com/dotnet/machinelearning/pull/3046#discussion_r268257594) [](ancestors = 268257594,268028098)